src/enums.h

```diff
@@ -237,7 +229,7 @@ enum ConditionParam_t {
 	CONDITION_PARAM_SUBID = 45,
 	CONDITION_PARAM_FIELD = 46,
 	CONDITION_PARAM_DISABLE_DEFENSE = 47,
-	CONDITION_PARAM_AGGRESSIVE = 54,
+	CONDITION_PARAM_AGGRESSIVE = 48,
```

Comment:
Revert the number please, there is a reason I left this like that. I have to add back support for criticals and leech for conditions